.github/workflows/package_for_test.yml

```diff
@@ -73,8 +79,19 @@ jobs:
           CSC_LINK: ${{ secrets.MAC_CERTIFICATE_BASE64 }}
           CSC_KEY_PASSWORD: ${{ secrets.MAC_CERTIFICATE_PASSWORD }}
 
+      - name: Package for MacOS for skip code sign
+        if: ${{ matrix.os == 'macos-latest' && env.APPLE_ID == '' }}
+        run: |
+          export CSC_IDENTITY_AUTO_DISCOVERY=false
```

Comment:
`CSC_IDENTITY_AUTO_DISCOVERY` can be set in `env` along with `SKIP_NOTARIZE`.
Another question is that is `CSC_IDENTITY_AUTO_DISCOVERY=false` necessary when `SKIP_NOTARIZE=true`?

Reply:
`CSC_IDENTITY_AUTO_DISCOVERY ` uses for code sign. And `SKIP_NOTARIZE` uses for notarizing.
https://developer.apple.com/documentation/security/code_signing_services
https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution